test/edu/washington/escience/myria/operator/UserDefinedAggregatorTest.java

```diff
@@ -197,7 +197,7 @@ public void testRowOfMax() throws Exception {
         assertEquals(2, result.numColumns());
         assertEquals(Type.LONG_TYPE, result.getSchema().getColumnType(0));
         assertEquals(Type.STRING_TYPE, result.getSchema().getColumnType(1));
-        assertEquals(9999, result.getLong(0, 0));
+        assertEquals(10000, result.getLong(0, 0));
```

Comment:
I think the old test was wrong since the first column is the row index, and the expression is `+1`, so the last row should have row index to be 10000. I didn't check why the old test was wrong though.